### PR TITLE
Adds Dissolvable Cleaner Tablets

### PR DESCRIPTION
**Pull request by HanSolo1519**
Adds Idris-themed space cleaner pills (to dissolve in water (or eat, I'm not your boss)) and boxes for them.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -1230,3 +1230,12 @@
...
 
+/obj/item/storage/box/spacecleaner
+	name = "Idris advamced dissolvable cleaner tablets"
+	desc = "A box of advanced chemical tablets designed by Idris Incorporated. A new generation of cleaning chemicals. Instructions on the box read; dissolve tablet fully in bucket of water, do not consume tablet, do not consume cleaner water from concentration."
+	icon = 'icons/obj/tabletcase.dmi'
+	icon_state = "tabletcase"
+	max_storage_space = 16
+	starts_with = list(
+		/obj/item/reagent_containers/pill/spacecleaner = 16,
+	)
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/storage/boxes.dm:1241]:
```suggestion
/obj/item/storage/box/cleaner_tablets
	name = "\improper Idris Incorporated cleaner tablets case"
	desc = "A box of advanced formula chemical tablets designed by Idris Incorporated."
	desc_extended = "A new generation of cleaning chemicals, according to Idris at least. The instructions on the box reads: \"Dissolve tablet fully in container of water\". A warning label mentions that you should not consume the tablets nor drink the mixture after dissolving them."
	icon = 'icons/obj/tabletcase.dmi'
	icon_state = "tabletcase"
	max_storage_space = 16
	starts_with = list(
		/obj/item/reagent_containers/pill/cleaner_tablet = 16
	)
```

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -333,3 +333,11 @@
 	desc = "A prophylactic antibiotic that kills infections before they start."
 	icon_state = "pill8"
 	reagents_to_add = list(/singleton/reagent/steramycin = 5)
+
+/obj/item/reagent_containers/pill/spacecleaner
+	name = "dissolvable cleaner tablet"
+	desc = "A highly advanced dissolvable tablet laden with the cutting edge of Idris chemical science. Smells like dried bleach. It's probably just bleach."
+	icon = 'icons/obj/cleanertablet.dmi'
+	icon_state = "cleanertablet"
+	reagents_to_add = list(singleton/reagent/spacecleaner = 15)
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/pill.dm:342]:
```suggestion
/obj/item/reagent_containers/pill/cleaner_tablet
	name = "dissolvable cleaner tablet"
	desc = "An advanced formula dissolvable tablet laden with the cutting edge of Idris chemical science. Smells like dried bleach. It's probably just bleach."
	icon = 'icons/obj/cleanertablet.dmi'
	icon_state = "cleanertablet"
	reagents_to_add = list(singleton/reagent/spacecleaner = 15)
```

**Comment by HanSolo1519** [comment]
Thanks gem you're a major help.

I still have to compile, test, and map the boxes into the janitorial closets (and probably junk loottable, now that I'm thinking about it). I'm having computer issues though so it may be a couple days.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
Don't make an entirely new .dmis for singular sprites, use the regular box and illustration system for the box and put the pill in chemical.dmi

**Comment by SleepyGemmy** [comment]
> I still have to compile, test, and map the boxes into the janitorial closets [...]

make sure you add it to the lockers in the code and not map them in.

**Comment by HanSolo1519** [comment]
> Don't make an entirely new .dmis for singular sprites, use the regular box and illustration system for the box and put the pill in chemical.dmi

I was explicitly told not to, for some reason. I'm not objected to changing, but I want to make sure that I'm actually supposed to after I asked and was told otherwise earlier.
image
image
image

**Comment by alsoandanswer** [comment]
> > Don't make an entirely new .dmis for singular sprites, use the regular box and illustration system for the box and put the pill in chemical.dmi
> 
> I was explicitly told not to, for some reason. I'm not objected to changing, but I want to make sure that I'm actually supposed to after I asked and was told otherwise earlier. image image image

Well, for clothes, they use contained sprites, same with custom items. So they have their own .dmis.

However these should be fine

**Comment by HanSolo1519** [comment]
Moved the pill icon to chemical.dm
Replaced Lizzy's sprite (Thank you for the help though!) with a normal box illustration in line with the rest of them.
Added a box of cleaner pills to the custodian locker startlist.
Added the pill and pillbox to the maintenance loottable (I think)

Still need to get build.bat to not error with windows when I go to test it, but other than that it should be good. I'm not going to finalize this PR until I can make sure it works though.

**Comment by NonQueueingMatt** [comment]
Reopen when conflicts are resolved. And make sure you tag your request as a finished one, not as a draft

**Pull request closed without merging**